api/api/fileProcessing/index.ts

```diff
@@ -406,4 +406,45 @@ export default function (app: Application) {
       });
     })
   );
+
+  /**
+   * @api {get} /api/v1/recordings/:id/tracks Get tracks for recording
+   * @apiName GetTracks
+   * @apiGroup Tracks
+   * @apiDescription Get all tracks for a given recording and their tags.
+   *
+   * @apiUse V1ResponseSuccess
+   * @apiSuccess {JSON} tracks Array with elements containing id,
+   * algorithm, data and tags fields.
+   *
+   * @apiUse V1ResponseError
+   */
+  app.get(
+    `${apiUrl}/:id/tracks`,
+    param("id").isInt().toInt(),
+
```

Comment:
Worth noting that there's not user authentication happening here to make sure the user has access to the recording - but I'm happy to fix this up as I merge into my branch.